backend/controller/controller.go

```diff
@@ -383,15 +383,23 @@ func (s *Service) GetSchema(ctx context.Context, c *connect.Request[ftlv1.GetSch
 	if err != nil {
 		return nil, err
 	}
-	sch := &schemapb.Schema{
-		Modules: slices.Map(schemas, func(d *schema.Module) *schemapb.Module {
-			return d.ToProto().(*schemapb.Module) //nolint:forcetypeassert
-		}),
+	modules := []*schemapb.Module{ //nolint:forcetypeassert
+		schema.Builtins().ToProto().(*schemapb.Module),
 	}
-	return connect.NewResponse(&ftlv1.GetSchemaResponse{Schema: sch}), nil
+	modules = append(modules, slices.Map(schemas, func(d *schema.Module) *schemapb.Module {
+		return d.ToProto().(*schemapb.Module) //nolint:forcetypeassert
+	})...)
+	return connect.NewResponse(&ftlv1.GetSchemaResponse{Schema: &schemapb.Schema{Modules: modules}}), nil
 }
 
 func (s *Service) PullSchema(ctx context.Context, req *connect.Request[ftlv1.PullSchemaRequest], stream *connect.ServerStream[ftlv1.PullSchemaResponse]) error {
+	if err := stream.Send(&ftlv1.PullSchemaResponse{ //nolint:forcetypeassert
+		Schema:     schema.Builtins().ToProto().(*schemapb.Module),
+		More:       true,
```

Comment:
Is it safe to have `true` here?  I had this change in the `watchModuleChanges` so that I could check the `initialCount`
```
	builtins := schema.Builtins().ToProto().(*schemapb.Module)
	buildinsResponse := &ftlv1.PullSchemaResponse{
		ModuleName: builtins.Name,
		Schema:     builtins,
		ChangeType: ftlv1.DeploymentChangeType_DEPLOYMENT_ADDED,
		More:       initialCount > 1,
	}
	sendChange(buildinsResponse)
```

Reply:
I wasn't 100% sure so I YOLOed it. If you've already got this change locally, just fold in the bits you don't have like the builtin rename, and merge it with your code.

Reply:
Ok will do!

Reply:
Pushed an update to this PR.